.gitignore

```
@@ -0,0 +1,4 @@
+node_modules
+dist
+./src/.DS_Store
+./src/images/.DS_Store
```

Можно лучше: группировать игнорируемые файлы.
Можно указать файл без пути, тогда он будет искаться во всех директориях:
```suggestion
.DS_Store
```

---

сделал, спасибо